src/plot_api/subroutines.js

```diff
@@ -306,8 +307,8 @@ function lsInner(gd) {
             yLinesXLeft = getLinePosition(ya, xa, 'left');
             yLinesXRight = getLinePosition(ya, xa, 'right');
 
-            extraSubplot = (!ya._anchorAxis || subplot !== xa._mainSubplot);
-            if(extraSubplot && ya.ticks && ya.mirror === 'allticks') {
+            extraSubplot = (!ya._anchorAxis || subplot !== ya._mainSubplot);
+            if(extraSubplot && (ya.mirror === 'allticks' || ya.mirror === 'all')) {
```

Comment:
Nice find! Is there a test :lock: ing this down?

Reply:
ha, there was a test (the subplot case) locking down the *changes* but not the parts that remained -> added these in https://github.com/plotly/plotly.js/pull/2936/commits/545ad472b8e8727e19208bac22fd568e17c781a2